### PR TITLE
Set resource req  in all the desired pods in the wanted namespaces

### PR DESCRIPTION
**Pull request by ITViking**
#### What does this PR do?
This PR adjusts the memory requests for the pods: cli, nginx, varnish and redis. This should help kubernetes to better allocate workloads

#### Should this be tested by the reviewer and how?
This can be run without any concerns. Please do so. 

#### Any specific requests for how the PR should be reviewed?
Test run it and read it through

#### What are the relevant tickets?
https://reload.atlassian.net/jira/software/c/projects/DDFDRIFT/boards/464?selectedIssue=DDFDRIFT-96

### CODE REVIEW

File: infrastructure/environments/dplplat01/sites.yaml [hunk 1]
```diff
@@ -439,6 +439,7 @@ sites:
       - nginx.main.kobenhavn.dplplat01.dpl.reload.dk
       - varnish.main.kobenhavn.dplplat01.dpl.reload.dk
     plan: webmaster
+    webmaster-cms-version: 2024.27.1
```
Comment (kasperg) [inline infrastructure/environments/dplplat01/sites.yaml:442]: This does not seem to be related to the change at hand?

File: infrastructure/task/scripts/adjust-resource-requests.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  # Skip system namespaces - those we have enough controll over
+  if [[ " ${SYSTEM_NAMESPACES[@]} " =~ " ${NS} " ]]; then
+    continue
+  fi
+  echo "##   $NS    ##"
+  # Pod to be adjusted
+  DEPLOYMENTS=("cli" "nginx" "varnish" "redis")
+
+  # Desired memory request
+  MEMORY_REQUEST="150Mi"
```
Comment (kasperg) [inline infrastructure/task/scripts/adjust-resource-requests.sh:45]: Consider if it worth noting what the value of this request is based on? Random higher value, experience/observations? Best practice?

File: infrastructure/dpladm/env-repo-template/standard/.lagoon.yml [hunk 3]
```diff
@@ -56,7 +56,13 @@ tasks:
         name: Import config translations
         command: drush dpl_po:import-remote-config-po da https://danskernesdigitalebibliotek.github.io/dpl-cms/translations/da.config.po
         service: cli
-
+    - run:
+        name: Set Pod resource requests and limits
+        command: |
+            set -e;
+            echo "setting resource requests for cli"
+            kubernetes patch deployment cli -n ${LAGOON_PROJECT_NAME} -p '{"spec":{"template":{"spec":{"containers":[{"name":"cli","resources":{"requests":{"memory":"20Mi"}}}]}}}'
```
Comment (kasperg) [inline infrastructure/dpladm/env-repo-template/standard/.lagoon.yml:64]: I wonder what this resource request is about.

- Why is is handled separately from `infrastructure/task/scripts/adjust-resource-requests.sh`?
- What is the origin of the `20Mi` value?

Comment (ITViking) [reply infrastructure/dpladm/env-repo-template/standard/.lagoon.yml]: Good catch - This should have been removed as it doesn't work

File: infrastructure/task/scripts/adjust-resource-requests.sh (hunk 2 above)
Comment (ITViking) [reply infrastructure/task/scripts/adjust-resource-requests.sh:45]: right now it is just based on a hunch. We need to get the exact numbers from grafana

**Comment by ITViking** [comment]
@kasperg vil du give denne her et kig mere?

File: infrastructure/environments/dplplat01/sites.yaml (hunk 1 above)
Comment (kasperg) [reply infrastructure/environments/dplplat01/sites.yaml:442]: I expect it should be removed then. It still shows off as a diff.

File: infrastructure/task/scripts/adjust-resource-requests.sh (hunk 2 above)
Comment (kasperg) [reply infrastructure/task/scripts/adjust-resource-requests.sh:46]: OK. Can we get this info from Grafana now given the data we have available there?

File: infrastructure/dpladm/bin/sync-site.sh [hunk 4]
```diff
@@ -34,6 +34,19 @@ function getSiteDplCmsRelease {
...
+    local wmRelease
+    wmRelease=$(yq eval ".sites.${1}.webmaster-cms-version" "${2}")
+    if [[ "${wmRelease}" == "null" ]]; then
+        echo ""
+        return
+    fi
+
+    echo "${wmRelease}"
+    return
+}
```
Comment (kasperg) [inline infrastructure/dpladm/bin/sync-site.sh:48]: This does not seem to be related to the change at hand?

File: infrastructure/dpladm/env-repo-template/standard/lagoon/cli.dockerfile [hunk 5]
```diff
@@ -7,6 +7,11 @@ FROM uselagoon/php-8.1-cli-drupal:${LAGOON_IMAGES_RELEASE_TAG}
 
 COPY --from=release /app /app
 RUN mkdir -p -v -m775 /app/web/sites/default/files
+# install kubectl - we need as long as we cant set the resource request
```
Comment (kasperg) [inline infrastructure/dpladm/env-repo-template/standard/lagoon/cli.dockerfile:10]: Appreciate you mentioning *why* this is needed.

**Review by kasperg — COMMENTED** [review]
I have not tried running this in practice.

It seems as if this PR needs to be separated from #403.

**Review by kasperg — APPROVED** [review]
